specs/core/0_beacon-chain.md

```diff
@@ -1340,7 +1344,7 @@ def initiate_validator_exit(state: BeaconState,
     if validator.status != ACTIVE:
         return
 
-    validator.status = ACTIVE_PENDING_EXIT
+    validator.status = exit_type
     validator.latest_status_change_slot = state.slot
 ```
 
```

Comment:
We need to move the `whistblower_reward` from `exit_validator` to `initiate_validator_exit`